streaming-binlog/src/test/java/org/openmrs/analytics/FhirConverterTest.java

```diff
@@ -125,4 +125,63 @@ public void shouldIgnoreEventWithUnknownTable() throws Exception {
 		Mockito.verify(openmrsUtil, Mockito.times(0)).fetchFhirResource(Mockito.anyString());
 	}
 	
+	@Test
+	public void shouldGenerateFhirResourcesForTablesThatHaveBeenMappedInConfig() {
+		Map<String, String> messageBody = DebeziumTestUtil.genExpectedBody();
+		
+		String tables[] = { "obs", "encounter", "cohort", "person", "provider", "relationship", "patient", "drug", "allergy",
+		        "order", "drug_order", "test_order", "program" };
+		
+		int i;
+		for (i = 0; i < tables.length; i++) {
```

Comment:
Can we do `for (String table : tables)` instead of using an index? (also below in the next two methods)

Reply:
Noted, thanks!